### PR TITLE
Assign NAME property in driver_internal.sh.

### PR DESCRIPTION
**Pull request by trironkk**

### CODE REVIEW

File: scripts/driver/driver_internal.sh [hunk 1]
```diff
@@ -56,22 +55,32 @@ echo "$parameters" \
...
 
-# use base64 for BSD systems where tr won't handle illegal characters
-export NAMESPACE="apptest-$(cat /dev/urandom \
-    | base64 \
-    | tr -dc 'a-z0-9' \
-    | fold -w 8 \
-    | head -n 1)"
-export NAME="$name"
+export NAME="name-$test_id"
+export NAMESPACE="namespace-$test_id"
```
Comment (vcanaa) [inline scripts/driver/driver_internal.sh]: This namespace doesn't tell what the namespace is used for. Please keep apptest or an equivalent naming.

File: scripts/driver/driver_internal.sh [hunk 2]
```diff
@@ -56,22 +55,32 @@ echo "$parameters" \
...
     --type NAMESPACE)"
 
-# use base64 for BSD systems where tr won't handle illegal characters
-export NAMESPACE="apptest-$(cat /dev/urandom \
-    | base64 \
-    | tr -dc 'a-z0-9' \
-    | fold -w 8 \
-    | head -n 1)"
-export NAME="$name"
+export NAME="name-$test_id"
```
Comment (vcanaa) [inline scripts/driver/driver_internal.sh]: Remove "name-"

File: scripts/driver/driver_internal.sh [hunk 3]
```diff
@@ -47,7 +47,6 @@ setup_kubeconfig.sh
 
 # Extract schema and values files.
 docker run \
```
Comment (vcanaa) [inline scripts/driver/driver_internal.sh]: Fun fact, if you don't include -t, you cant ctrl+c this.

Comment (trironkk) [reply scripts/driver/driver_internal.sh]: Needs a ctrl-d, i believe.

File: scripts/driver/driver_internal.sh (hunk 1 above)
Comment (trironkk) [reply scripts/driver/driver_internal.sh]: Done.

File: scripts/driver/driver_internal.sh (hunk 2 above)
Comment (trironkk) [reply scripts/driver/driver_internal.sh]: Done.

**Pull request merged**